react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+export default function HeatButton() {
+  const [clickIndex, setClickIndex] = useState(0);
+
+  function handleClick() {
+    setClickIndex(clickIndex + 1);
+    console.log('clicks:', clickIndex + 1);
+  }
+
+const fontColor = clickIndex >= 12 ? "black" : "white";
+let buttonColor;
+
+  switch(true) {
```

Comment:
Another `switch` lover, huh?

I hadn't seen a switch used like this until Josh used it in the last cohort. Where did you both learn this technique?